### ISSUE TITLE
Keyerror for a CNV

### ISSUE THREAD

**Issue by tsladecek** [Genotek/ClassifyCNV#5]
Hi,

I am trying to evaluate the following CNV (hg38):
`chr16   34963121  36048843 DUP`

I ran the `update_clingen.sh` script and I am using the latest ClassifyCNV version (1.1.0)

However, the script crashes with this error message:
```
ClassifyCNV-1.1.0/ClassifyCNV.py Version 1.1.0
Traceback (most recent call last):
  File "ClassifyCNV-1.1.0/ClassifyCNV.py", line 817, in <module>
    dosage_sensitivity()
  File "ClassifyCNV-1.1.0/ClassifyCNV.py", line 657, in dosage_sensitivity
    assign_dup_points_s2(dosage_res_dup)
  File "ClassifyCNV-1.1.0/ClassifyCNV.py", line 462, in assign_dup_points_s2
    for gene in cnv_genes[cnv]:
KeyError: 'chr16_34963121_36048843_DUP'
```

Best,
Tomas

**Comment by tgurbich** [comment]
Hi Tomas,

Thank you very much for reporting this. I fixed this bug and released version 1.1.1 which should no longer produce an error.

Best,
Tatiana

**Comment by tsladecek** [comment]
Thank you for the quick response, it is working now :)
